experimental/segmenter/src/dictionary.rs

```diff
@@ -197,20 +216,7 @@ mod tests {
 
     #[test]
     fn cj_dictionary_test() {
-        // This test data is created by the following using ICU4C tools
-        // LD_LIBRARY_PATH=lib bin/gendict --uchars data/brkitr/dictionaries/cjdict.txt tmp.bin
-        // dd if=tmp.bin of=cjdict.dict bs=1 skip=64
-        const CJ_DICTIONARY: &ZeroSlice<u16> = match ZeroSlice::<u16>::try_from_bytes(
-            include_bytes!("../tests/testdata/cjdic.dict"),
-        ) {
-            Ok(s) => s,
-            Err(_) => panic!("invalid dictionary data"),
-        };
-
-        let data = UCharDictionaryBreakDataV1 {
-            trie_data: CJ_DICTIONARY.as_zerovec(),
-        };
-        let payload = DataPayload::<UCharDictionaryBreakDataV1Marker>::from_owned(data);
+        let payload = get_payload("ja").unwrap();
```

Comment:
This is a good question (how to represent the CJK dictionary as a language tag). Let me think about it more.